components/SidebarToggle.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <button
-    class="m-1 h-min rounded-full bg-fog px-3 py-2 hover:bg-smoke dark:bg-basalt hover:dark:bg-granite sm:hidden md:px-4 md:py-3"
+    class="flex h-10 w-10 items-center justify-center rounded-full bg-fog hover:bg-smoke dark:bg-basalt hover:dark:bg-granite"
   >
-    <Icon name="heroicons:bars-3" class="text-center" />
+    <Icon name="heroicons:bars-3" />
```

Comment:
Fixed some existing issues with the vertical centring of the icon within the button